addons/l10n_ch_sale/tests/test_swissqr.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+from odoo.tools.misc import mod10r
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestSwissQRSaleOrder(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='l10n_ch.l10nch_chart_template'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+    def setUp(self):
+        super(TestSwissQRSaleOrder, self).setUp()
+
+        self.env['ir.config_parameter'].create({'key': 'l10n_ch.print_qrcode', 'value': '1'})
+        self.customer = self.env['res.partner'].create({
```

Comment:
From 16.0 and on you can use the customer 'Easy Clean Lausanne' to test anything Swiss-QR related :)
CH company is also correctly configured in the demo data